osu.Game/Screens/Edit/Compose/Components/DistanceSnapGrid.cs

```diff
@@ -23,7 +23,9 @@ public abstract class DistanceSnapGrid : CompositeDrawable
         /// <summary>
         /// The spacing between each tick of the beat snapping grid.
         /// </summary>
-        protected float DistanceSpacing { get; private set; }
+        protected float DistanceBetweenTick { get; private set; }
```

Comment:
Probably should be renamed to `DistanceBetweenTicks` (as in, plural ticks). "between tick" sounds a touch weird.